integration-tests/src/test/scala/za/co/absa/spline/CobrixWriteSpec.scala

```diff
@@ -0,0 +1,108 @@
+      |			         REDEFINES  COMPANY-ID-NUM PIC X(3).
+      |           05  METADATA.
+      |               10  CLIENTID              PIC X(15).
+      |               10  REGISTRATION-NUM      PIC X(10).
+      |               10  NUMBER-OF-ACCTS       PIC 9(03) COMP-3.
+      |               10  ACCOUNT.
+      |                   12  ACCOUNT-DETAIL    OCCURS 80
+      |                                         DEPENDING ON NUMBER-OF-ACCTS.
+      |                      15  ACCOUNT-NUMBER     PIC X(24).
+      |                      15  ACCOUNT-TYPE-N     PIC 9(5) COMP-3.
+      |                      15  ACCOUNT-TYPE-X     REDEFINES
+      |                           ACCOUNT-TYPE-N  PIC X(3).
+      |
+      |""".stripMargin
+
+  val bytes: Array[Byte]  = Array[Byte](
+    0x00.toByte, 0x06.toByte, 0xC5.toByte, 0xE7.toByte, 0xC1.toByte, 0xD4.toByte, 0xD7.toByte, 0xD3.toByte,
+    0xC5.toByte, 0xF4.toByte, 0x40.toByte, 0x40.toByte, 0x00.toByte, 0x00.toByte, 0x0F.toByte, 0x40.toByte,
+    0x40.toByte, 0x40.toByte, 0x40.toByte, 0x40.toByte, 0x40.toByte, 0x40.toByte, 0x40.toByte, 0x40.toByte,
+    0x40.toByte, 0x40.toByte, 0x40.toByte, 0x40.toByte, 0x40.toByte, 0x40.toByte, 0x40.toByte, 0x40.toByte,
+    0x40.toByte, 0x40.toByte, 0x40.toByte, 0x40.toByte, 0x40.toByte, 0x40.toByte, 0x40.toByte, 0x40.toByte,
+    0x00.toByte, 0x3F.toByte, 0xF0.toByte, 0xF0.toByte, 0xF0.toByte, 0xF0.toByte, 0xF0.toByte, 0xF0.toByte,
+    0xF0.toByte, 0xF0.toByte, 0xF0.toByte, 0xF0.toByte, 0xF0.toByte, 0xF0.toByte, 0xF0.toByte, 0xF0.toByte,
+    0xF2.toByte, 0xF0.toByte, 0xF0.toByte, 0xF0.toByte, 0xF4.toByte, 0xF0.toByte, 0xF0.toByte, 0xF0.toByte,
+    0xF1.toByte, 0xF2.toByte, 0x00.toByte, 0x00.toByte, 0x0F.toByte, 0xF0.toByte, 0xF0.toByte, 0xF0.toByte,
+    0xF0.toByte, 0xF0.toByte, 0xF0.toByte, 0xF0.toByte, 0xF0.toByte, 0xF0.toByte, 0xF0.toByte, 0xF0.toByte,
+    0xF0.toByte, 0xF0.toByte, 0xF0.toByte, 0xF3.toByte, 0xF0.toByte, 0xF0.toByte, 0xF0.toByte, 0xF4.toByte,
+    0xF0.toByte, 0xF0.toByte, 0xF1.toByte, 0xF0.toByte, 0xF2.toByte, 0x00.toByte, 0x00.toByte, 0x1F.toByte,
+    0xF0.toByte, 0xF0.toByte, 0xF0.toByte, 0xF0.toByte, 0xF0.toByte, 0xF0.toByte, 0xF0.toByte, 0xF0.toByte,
+    0xF5.toByte, 0xF0.toByte, 0xF0.toByte, 0xF6.toByte, 0xF0.toByte, 0xF0.toByte, 0xF1.toByte, 0xF2.toByte,
+    0xF0.toByte, 0xF0.toByte, 0xF3.toByte, 0xF0.toByte, 0xF1.toByte, 0xF0.toByte, 0xF0.toByte, 0xF0.toByte,
+    0x00.toByte, 0x00.toByte, 0x2F.toByte
+  )
+
+  val bos = new FileOutputStream(cobrixPath)
+  bos.write(bytes)
+  bos.close()
+
+  it should "support Cobrix as a source" in
+    withNewSparkSession(spark => {
+      withLineageTracking(spark)(lineageCaptor => {
+
+        val (plan1, _) = lineageCaptor.lineageOf(spark
+          .read
+          .format("cobol")
+          .option("copybook_contents", copybook)
+          .option("is_record_sequence", "true")
+          .option("schema_retention_policy", "collapse_root")
+          .option("segment_id_" +
```

Comment:
very minor thing (sorry for bothering) - unnecessary string concatenation. Also we prefer string interpolation over concatenation, just for the info.
Overwise LGTM

Reply:
My bad. Updating now